bindings/rust/src/test_formats/compute_blob_kzg_proof.rs

```diff
@@ -6,14 +6,21 @@ use serde::Deserialize;
 #[derive(Deserialize)]
 pub struct Input<'a> {
     blob: &'a str,
+    commitment: &'a str,
 }
 
 impl Input<'_> {
-    pub fn get_blob(&self) -> Result<Box<Blob>, Error> {
+    pub fn get_blob(&self) -> Result<Blob, Error> {
         let hex_str = self.blob.replace("0x", "");
         let bytes = hex::decode(hex_str).unwrap();
```

Comment:
If the test format specifies a 0x prefixed string, then we could also do 
`hex::decode(self.blob[2..])` to avoid creating a new string.